src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(n -> n.split(COMA))
+                .flatMap(Arrays::stream)
```

Comment:
you can replace it with one operation
```suggestion
                .flatMap(n -> Arrays.stream(n.split(COMA)))
```